conda_forge_tick/auto_tick.xsh

```diff
@@ -66,8 +66,20 @@ def run(attrs, migrator, feedstock=None, protocol='ssh',
                                    fork=fork,
                                    gh=gh)
 
-    # migrate the `meta.yaml`
     recipe_dir = os.path.join(feedstock_dir, 'recipe')
+    # if postscript/activate no noarch
+    suffixs = ['pre-unlink', 'post-link', 'pre-link']
```

Comment:
You can add `activate` here

Reply:
And maybe change the name `suffixs` since they aren't suffixes now.